documentation/book/con-kafka-connect-external-configuration.adoc

```diff
@@ -0,0 +1,94 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-kafka-connect-external-configuration.adoc
+
+[id='con-kafka-connect-external-configuration-{context}']
+
+= Storing connector configurations externally
+
+You can mount ConfigMaps or Secrets into a Kafka Connect pod as volumes or environment variables.
+Volumes and environment variables are configured in the `externalConfiguration` property in `KafkaConnect.spec` and `KafkaConnectS2I.spec`.
+
+== External configuration as environment variable
+
+The `env` property is used to specify one or more environment variables.
+These variables can contain a value from either a ConfigMap or a Secret.
+
+NOTE: The names of user-defined environment variables cannot start with `KAFKA_` or `STRIMZI_`.
+
+To mount a value from a Secret to an environment variable, use the `valueFrom` property and the `secretKeyReference` as shown in the following example.
+
+.Example of environment variable set to a value from a Secret
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: KafkaConnect
+metadata:
+  name: my-connect
+spec:
+  # ...
+  externalConfiguration:
+    env:
+      - name: MY_ENVIRONMENT_VARIABLE
+        valueFrom:
+          secretKeyRef:
+            name: my-secret
+            key: my-key
+----
+
+A common use case for mounting Secrets into environment variables is when your connector needs to communicate with Amazon AWS and needs to read the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables with credentials.
```

Comment:
into environment variables > to environment variables

(I missed this in the first review.)